src/lib/isInDrawer.js

```diff
@@ -0,0 +1,9 @@
+export default function isInDrawer() {
+  const result = false
+  // Need try/catch because in server mode this is not available.
```

Comment:
This means we might see flickering, because the server rendered page will always have a footer/header. I wonder if there is a solution here aside from having separate URLs? 🤔

Reply:
Another approach is to do this with a css classname on the body depending on the url param (which I assume can be rendered server side).

```html
<body class="drawer-mode">
```
```css
.drawer-mode {
  .footer, header {
    display: none;
  }
}

Reply:
![image](https://user-images.githubusercontent.com/289488/52710611-69ea2d80-2f98-11e9-808d-0fa60d485423.png)

Reply:
> Another approach is to do this with a css classname on the body depending on the url param (which I assume can be rendered server side).

Good idea 👌 Although doesn't solve the flickering.

Reply:
Why would there be flickering? I would expect the server to render the `.wrapper` without `.drawer-mode` since the try/catch fails. Do you mean that a separate JS would run subsequently?

Reply:
Exactly: the server will render it without `.drawer-mode` so the first HTML the client receives will have a header/footer, but then JS will run and hide the header.

Reply:
Perhaps the best approach would be to adjust the 'mobile' version:
* Header to take up minimal space
* Hides table of contents.
* Relocate search bar.
* Optimize footer content.

Reply:
It's a good idea, but I would still prefer to hide those in the drawer.